test/openbci-sdk-test.js

```diff
@@ -969,22 +755,16 @@ describe('#impedanceTesting', function() {
                 impedanceArray[0].channel.should.be.equal(1);
             });
             describe('#inputP', function() {
-                it('data array NOT present',function() {
-                    impedanceArray[0].P.data.length.should.be.equal(0);
-                });
-                it('valid average',function() {
-                    impedanceArray[0].P.should.have.property('average').equal(-1);
+                it('got raw impedance value',function() {
+                    impedanceArray[0].P.should.have.property('raw').equal(-1);
```

Comment:
is there a doc section where you explain why you expect the values you're testing against?